### DISCUSSION TITLE
Dark mode based on the Apollo Midnight theme

### DISCUSSION THREAD

**Discussion by hwillson** [apollographql/apollo-client-devtools#411] [Ideas]
It would be awesome to wire up full dark mode support, leveraging elements from [Apollo's Midnight theme](https://github.com/apollographql/apollo-midnight).

@daniman @jglovier - this won't make it into 3.0, but I'm adding it here so we don't lose sight of it. Thanks for the suggestion!

**Comment by jerelmiller** [comment]
With the update to Apollo's new Orbit theme in https://github.com/apollographql/apollo-client-devtools/pull/1180, dark and light modes are now supported using this theme. As such, I'm going to go ahead and close this as completed 🙂
